packages/cli/templates/vue3/src/style.css

```diff
@@ -64,12 +64,46 @@ body {
 
 /*Layout Styles*/
 
-.field-plugin {
+h2 {
+  font-size: 0.875rem;
+  line-height: 1.125rem;
+  font-weight: 500;
+  margin: 1rem 0;
+}
+
+.counter-value {
+  margin-top: 1.25rem;
+  margin-bottom: 1rem;
+  font-size: 1.625rem;
+  font-weight: 700;
+  text-align: center;
+}
+
+.btn {
+  font-size: 0.875rem;
+  font-weight: 500;
+  padding: 0.8125rem 1rem;
+  background: transparent;
+  border: 1px solid var(--light);
+  border-radius: 0.3125rem;
+  cursor: pointer;
+}
+
+.w-full {
+  width: 100%;
+}
+
+.btn:hover {
+  background-color: var(--light_75);
+}
+
+/* .field-plugin {
```

Comment:
Why is this commented out?

Reply:
I should've deleted that. Applying the design, I had to rewrite the css, and that was the old code.